src/inc/regdisp.h

```diff
@@ -53,6 +53,11 @@ struct REGDISPLAY_BASE {
 #endif // !_TARGET_X86_ && !_TARGET_ARM
 };
 
+inline PCODE GetControlPC(REGDISPLAY_BASE *pRD) {
```

Comment:
I'm not sure. It may block cross-architecture crossgen (which may be required for ARM pipeline bring up).